ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/ClusterStatusResource.java

```diff
@@ -71,12 +71,17 @@ private ClusterStatusResponse getResponse() {
             entry -> new KsqlHostEntity(entry.getKey().host(), entry.getKey().port()) ,
             entry -> new HostStatusEntity(entry.getValue().isHostAlive(),
                                           entry.getValue().getLastStatusUpdateMs(),
-                                          lagReportingAgent.isPresent()
-                                              ? lagReportingAgent.get().getLagPerHost(
-                                                  entry.getKey())
-                                              : EMPTY_HOST_STORE_LAGS)
+                                          getHostStoreLags(entry.getKey()))
         ));
 
     return new ClusterStatusResponse(response);
   }
+
+  private HostStoreLags getHostStoreLags(final KsqlHost ksqlHost) {
+    if (!lagReportingAgent.isPresent()) {
+      return EMPTY_HOST_STORE_LAGS;
+    }
+    final Optional<HostStoreLags> lags = lagReportingAgent.get().getLagPerHost(ksqlHost);
+    return lags.orElse(EMPTY_HOST_STORE_LAGS);
+  }
```

Comment:
Think this can just be:

```java
private HostStoreLags getHostStoreLags(final KsqlHost ksqlHost) {
    return lagReportingAgent
        .map(agent -> agent.getLagPerHost(ksqlHost))
        .orElse(EMPTY_HOST_STORE_LAGS);
  }
```

Reply:
Done